### DISCUSSION TITLE
Certificat Relay SMTP error

### DISCUSSION THREAD

**Discussion by bossdu15000** [Mailu/Mailu#3028] [User support]
## Environment & Version

- `docker compose version`
- Version: 2.0

## Description

incomprehensible certificate problem:
`server certificate verification failed for mx-mibc-fr-03.mailinblack.com[40.89.154.15]:25: num=62:hostname mismatch`

verification :
`openssl s_client -starttls smtp -showcerts -connect mx-mibc-fr-03.mailinblack.com:25 -servername mx-mibc-fr-03.mailinblack.com | grep subject`

result :
`subject=/CN=*.mailinblack.com`

why this "hostname mismatch" error

**Comment by ghostwheel42** [comment]
Which component of mailu logged the message you sent?
`server certificate verification failed for mx-mibc-fr-03.mailinblack.com`
